collect_app/src/main/java/org/odk/collect/android/utilities/ToastUtils.java

```diff
@@ -0,0 +1,19 @@
+package org.odk.collect.android.utilities;
+
+import android.widget.Toast;
+
+import org.odk.collect.android.application.Collect;
+
+/**
+ * Created by shobhit on 6/3/17.
+ */
+
+public class ToastUtils {
+    public static void shortDuration(String message){
+        Toast.makeText(Collect.getInstance(),message, Toast.LENGTH_SHORT).show();
```

Comment:
OK with me. It’s sort of implied by the “ToastUtils.” but it would add clarity for someone who statically imported the methods.